### ISSUE TITLE
Feature request: layer as a style option

### ISSUE THREAD

**Issue by akvadrako** [emersion/mako#210]
I would like important notifications to be displayed with layer=overlay and other notifications with layer=top.

**Comment by vilhalmer** [comment]
Duplicate of #183, though the title of this one is clearer. See also #123, #200 for other things that would require splitting apart notifications onto multiple layer surfaces.

**Comment by vilhalmer** [comment]
This will be possible after #228.

**Comment by emersion** [comment]
Fixed in #228
